scripts/build-testing-app.sh

```diff
@@ -27,7 +27,8 @@ npm install --save \
   webpack \
   webpack-cli \
   graphql \
-  graphql-tag
+  graphql-tag \
+  offix-cache
```

Comment:
Workaround for running tests against `latest` tag. Can be removed once we are able to test `master` tag of components without building them first.